sql/mysql/inspect.go

```diff
@@ -596,6 +608,14 @@ func (i *inspect) marDefaultExpr(c *schema.Column, x string) schema.Expr {
 	return &schema.RawExpr{X: x}
 }
 
+func (i *inspect) querySchema(ctx context.Context, query string, s *schema.Schema) (*sql.Rows, error) {
+	args := []interface{}{s.Name}
+	for _, t := range s.Tables {
+		args = append(args, t.Name)
+	}
+	return i.QueryContext(ctx, fmt.Sprintf(query, strings.Repeat("?, ", len(s.Tables)-1)+"?"), args...)
```

Comment:
`strings.Repeat("?, ", len(s.Tables)-1)+"?"), args...)`

Put in function. I see this several times.

Reply:
👍